packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -45,58 +44,68 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
   pseAccount: string;
   outcomeAddress: string;
 
-  static torrentUpdatedEventPrefix: string = 'torrentUpdated';
-
   constructor(opts: WebTorrent.Options & Partial<PaidStreamingExtensionOptions> = {}) {
-    super(opts);
+    super({tracker: {announce: defaultTrackers}, ...opts});
     this.peersList = {};
     this.pseAccount = opts.pseAccount;
     this.paymentChannelClient = opts.paymentChannelClient;
     this.outcomeAddress = opts.outcomeAddress;
   }
 
+  /** Enable the client capabilities to seed or leech torrents, enabling the paymentChannelClient
+   * and setting key information used to seed and leech torrents  */
   async enable() {
     if (!this.pseAccount || !this.outcomeAddress) {
       await this.paymentChannelClient.enable();
       this.pseAccount = this.paymentChannelClient.mySigningAddress;
-      log.info({pseAccount: this.pseAccount}, 'set pseAccount to sc-wallet signing address');
       this.outcomeAddress = this.paymentChannelClient.myEthereumSelectedAddress;
+      this.tracker.getAnnounceOpts = () => ({pseAccount: this.pseAccount});
+
+      log.info({pseAccount: this.pseAccount}, 'PSEAccount set to sc-wallet signing address');
       log.info(
         {outcomeAddress: this.outcomeAddress},
-        'set outcomeAddress to sc-wallet web3 wallet address'
+        'OutcomeAddress set to sc-wallet Wallet address'
       );
-      this.tracker.getAnnounceOpts = () => ({pseAccount: this.pseAccount});
     }
   }
 
+  /** Utility function. Disables the client capabilities to seed or leech torrents  */
   async disable() {
     log.warn('Disabling WebTorrentPaidStreamingClient');
     this.pseAccount = null;
     this.outcomeAddress = null;
   }
 
+  /**
+   * From a string, a file, a buffer, or an array from the previous, it creates a torrent,
+   * adds it to the client to be shared and announces it to the tracker.
+   */
   seed(
     input: WebTorrentSeedInput,
     optionsOrCallback?: TorrentOptions | TorrentCallback,
```

Comment:
Should we get rid of the complexity of optionsOrCallback and create a function signature that matches our needs?

Reply:
I decided to just match the function signature of the original function. Maybe we could refactor those functions in another PR?